nms/packages/magmalte/fbc_js_core/ui/stories/mui-components/appSideBar.stories.js

```diff
@@ -1,76 +0,0 @@
-/**
- * Copyright 2020 The Magma Authors.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * @flow
- * @format
- */
-
-import AppSideBar from '../../components/layout/AppSideBar';
```

Comment:
Wow I never even knew we were using storybook

Reply:
Yes, but it isn't running anymore